src/transformer/utils/get_verb.php

```diff
@@ -18,6 +18,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
+use src\transformer\utils as utils;
```

Comment:
This namespace is the same as where our function have, so I am not sure why we need this to add.

Reply:
I added it to stay consistent with the style used in the rest of the plugin.
https://github.com/xAPI-vle/moodle-logstore_xapi/blob/master/src/transformer/utils/get_multichoice_definition.php